portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,17 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+function addRandomFact() {
+  const facts =
+      ['I have over 20 pairs of sunglasses (that was a weird phase).', 'I can still recite most of Hamlet\'s third soliloquy.', 'I have 7 pets.', 'I can dance a jig.'];
```

Comment:
Best practice for long lines like this is to break it up over multiple lines. You can put each fact on a new line which will make them really easy to read in code

Reply:
Got it! I've got some long lines in my html file, should I be doing this there too?

Reply:
It depends... typically, I would keep an entire tag together on one line. So something like this

<div>Small Amount of Text</div>

is fine, but if you have more text, you might want to do 

<div>
    Large amount of text
</div>

And if you have a ton of text, you could do

<div>
    line 1
    line 2
</div>

It just depends on what sort of information is between the tags. I would try to keep each open or close tag on its own line, if possible. If the tag is getting too big though, I would follow something in here for guidance: https://stackoverflow.com/questions/44944936/indenting-html-tags-on-multiple-lines